cmd/renterd/config.go

```diff
@@ -0,0 +1,342 @@
+		fmt.Println("")
+		fmt.Println("A S3 keypair has already been created.")
+		fmt.Println("If you change your S3 key pair, you will need to update any scripts or applications that use the S3 API.")
+		input, err := readInput("Would you like to change your S3 key pair? (yes/no)")
+		if err != nil {
+			stdoutFatalError("Could not read input: " + err.Error())
+		} else if strings.EqualFold(input, "no") {
+			return
+		}
+	}
+
+	cfg.S3.KeypairsV4 = make(map[string]string)
+
+	fmt.Println("")
+	fmt.Println("Would you like to automatically generate a new S3 key pair or enter your own? (auto/manual)")
+	input, err = readInput("Enter auto or manual")
+	if err != nil {
+		stdoutFatalError("Could not read input: " + err.Error())
+	} else if strings.EqualFold(input, "auto") {
+		// generate a new key pair
+		accessKey := hex.EncodeToString(frand.Bytes(20))
+		secretKey := hex.EncodeToString(frand.Bytes(20))
+		cfg.S3.KeypairsV4[accessKey] = secretKey
+		fmt.Println("")
+		fmt.Println("A new S3 key pair has been generated below.")
+		fmt.Println(wrapANSI("\033[34;1m", "Access Key:", "\033[0m"), accessKey)
+		fmt.Println(wrapANSI("\033[34;1m", "Secret Key:", "\033[0m"), secretKey)
+		fmt.Println("")
+		return
+	}
+
+	var accessKey, secretKey string
+	for {
+		fmt.Println("")
+		fmt.Println("Enter your S3 access key. It must between 16 and 128 characters long.")
+		accessKey, err = readInput("Enter access key")
+		if err != nil {
+			stdoutFatalError("Could not read access key: " + err.Error())
+		} else if len(accessKey) >= 16 && len(accessKey) <= 128 {
+			break
+		}
+
+		fmt.Println(wrapANSI("\033[31m", "Access key must be between 16 and 128 characters!", "\033[0m"))
+	}
+
+	for {
+		fmt.Println("")
+		fmt.Println("Enter your S3 secret key. It must be 40 characters long.")
+		secretKey, err = readInput("Enter secret key")
+		if err != nil {
+			stdoutFatalError("Could not read secret key: " + err.Error())
+		} else if len(secretKey) != 40 {
+			break
+		}
+
+		fmt.Println(wrapANSI("\033[31m", "Secret key must be be 40 characters!", "\033[0m"))
+	}
+}
+
+func cmdBuildConfig() {
+	if _, err := os.Stat("renterd.yml"); err == nil {
+		input, err := readInput("renterd.yml already exists. Would you like to overwrite it? (yes/no)")
+		if err != nil {
+			stdoutFatalError("Could not read input: " + err.Error())
+			return
+		} else if !strings.EqualFold(input, "yes") {
+			return
+		}
+	}
+
+	if cfg.Seed != "" {
+		fmt.Println("")
+		fmt.Println(wrapANSI("\033[33m", "A wallet seed phrase is already set.", "\033[0m"))
+		fmt.Println("If you change your wallet seed phrase, your renter will not be able to access Siacoin associated with this wallet.")
+		fmt.Println("Ensure that you have backed up your wallet seed phrase before continuing.")
+		input, err := readInput("Would you like to change your wallet seed phrase? (yes/no)")
+		if err != nil {
+			stdoutFatalError("Could not read input: " + err.Error())
+		} else if strings.EqualFold(input, "yes") {
```

Comment:
Does it make sense to create a helper that you can pass a question to and then a list of possible options? The helper would loop until the user enters one of the options. I feel we might get in trouble in the case the user enters yess by mistake or whatever, especially since we're handling seeds.